fvtr/python/python.exp

```diff
@@ -105,6 +105,10 @@ set env(TMPDIR) "$FULLPATH/tmp[pid]"
 exec mkdir $env(TMPDIR)
 printit "TMPDIR=\"$env(TMPDIR)\""
 
+# Test for the issue #45
+printit "Executing sqlite3 importing test."
+exec $env(AT_DEST)/bin/python3 -c "import sqlite3"
```

Comment:
Isn't it possible to use test_sqlite?  It's already provided by the Python testsuite.
Besides that, are you sure this testcase will fail if the previous command fail?